src/app/article/page.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
```

Comment:
If a user refreshes this page then there is an error appears. Please fix it as you see suitable

Reply:
set last opened article in LocalStorage and get it if there is no context is ok?)